moira/cache.py

```diff
@@ -1,9 +1,31 @@
 from functools import wraps
 
 from twisted.internet import reactor, defer
+from collections import OrderedDict
 
-CACHE = {}
+#CACHE = {}
+class CacheClass(OrderedDict):
+    """
+    Autocleaning dict.
```

Comment:
Why call it "CacheClass" and write a comment instead of calling it "AutocleaningCacheClass"?